web-app/README.md

```diff
@@ -4,13 +4,43 @@ PWA client for event app, located at `app.$DOMAIN`.
 
 ## Getting started
 
+### Using Tailwind
+
+[Tailwind](https://tailwindcss.com/) is a collection 
+of [CSS utility classes](https://tailwindcss.com/docs/utility-first)
+but can also be composed into component classes for commonly used
+groups of styles like so
+```css
+/* web-app/static/tailwind.config.css */
+@tailwind base;
+
+@tailwind components;
+
+/*
+ Classes like this should be added directly after the @tailwind components
+ directive to avoid specificity issues.
+*/
+.my-awesome-button {
+  @apply bg-blue-500 text-white font-bold py-2 px-4 rounded;
+}
+
+@tailwind utilities;
+```
+
+Any time you change `web-app/static/tailwind.config.css`,
```

Comment:
There's nothing special about the name `tailwind.config.css`, I just picked it because it felt like a config file